frontend/app/dashboard/src/views/dashboard/webshop/edit/products/EditProductView.vue

```diff
@@ -83,6 +83,19 @@
         </template>
 
         <hr><STList>
+            <STListItem v-if="!patchedProduct.prices.some(p => p.uitpasBaseProductPriceId !== null) && (productPricesAvailableForUitpasBaseProductPrice.length !== 0 /*should never be the case*/)" :selectable="true" element-name="button" @click="addProductPrice(true)">
```

Comment:
```suggestion
            <STListItem v-if="!patchedProduct.prices.some(p => p.uitpasBaseProductPriceId !== null" :selectable="true" element-name="button" @click="addProductPrice(true)">
```
Hou het simpel of verplaats dit naar een computed property voor de leesbaarheid